src/events/presenceUpdate.ts

```diff
@@ -26,6 +27,21 @@ export default async function PresenceUpdateEvent(discordClient: DiscordClient,
 	}
 
 	const whiteListed = await CheckGameWhitelisted(streamingActivity);
+	
+	if ((serverSettings.streamLiveRole !== null || serverSettings.streamShout !== null) && whiteListed) {
```

Comment:
Should check if `streamTimeout > 0`, right?

Reply:
Not necessarily, but it would be a good optimization regardless